test/parallel/test-whatwg-url-constructor.js

```diff
@@ -142,3 +142,11 @@ function runURLSearchParamTests() {
 runURLSearchParamTests()
 runURLConstructorTests()
 /* eslint-enable */
+
+const message = {
+  code: 'ERR_INVALID_URL',
+  name: 'TypeError',
+  message: 'Invalid URL',
+};
+
+assert_throws(() => new URL('https://256.256.256.256.256'), message);
```

Comment:
We don't want to validate the message property of internal errors, as those are not bound to semver rules, see https://github.com/nodejs/node/blob/fc485a9cbe552902e2c1880210f83170d9d951aa/doc/contributing/writing-tests.md#L315-L325

Also, `assert_throws` has a different signature than `assert.throws`.

```suggestion
const assert = require('assert');
assert.throws(() => new URL('https://256.256.256.256.256'), { code: 'ERR_INVALID_URL' });
```

or

```suggestion
assert_throws(TypeError(), () => new URL('https://256.256.256.256.256'));
```

Reply:
Thanks for the reference! I'll update my pull request.